src/modals/ConnectToWalletModal.js

```diff
@@ -137,6 +151,22 @@ class ConnectToWalletModal extends React.Component {
                   </div>
                 </WalletTypeDiv>
               </Col>
+              <Col span={12}>
```

Comment:
Could we change UI to a row? Currently, it's two rows and the second row has only one wallet icon.

Reply:
Simply change span={12} to span={8}

Reply:
I made a change.